op-challenger/fault/service.go

```diff
@@ -31,7 +33,10 @@ type service struct {
 
 // NewService creates a new Service.
 func NewService(ctx context.Context, logger log.Logger, cfg *config.Config) (*service, error) {
-	client, err := ethclient.Dial(cfg.L1EthRpc)
+	// These hard-coded values will need to be moved into the config.
+	maxRetries := 5
+	retryInterval := time.Second * 5
+	client, err := client.DialEthClientWithRetry(ctx, cfg.L1EthRpc, backoff.Exponential(), maxRetries, retryInterval)
```

Comment:
Hmm this should use a linear backoff strategy? On start we dont want it to take longer than necessary